### ISSUE TITLE
Grader CLI output-json option doesn't work?

### ISSUE THREAD

**Issue by adhameer** [ocaml-sf/learn-ocaml#193]
I can't seem to get the output-json option for the grader CLI to work properly:

```
> learn-ocaml grade --output-json=.
Learnocaml v.0.8 running.
.                              - Success - 40 points
[ERROR] Could not load exercise at .: Unix.Unix_error(Unix.EISDIR, "open", ".")
```

Possibly I'm not using it right? It requires that the input be a directory and not a filename, and I've tried giving it different directories to write the results to with no success.

**Comment by AltGr** [comment]
Ah indeed, it's a bug, the option was set to a directory (to allow for grading multiple exercise), but the code still expected a file. I'll fix it shortly, thanks for reporting.

**Comment by AltGr** [comment]
Fixed by 9f59a055 (part of #189)
